### PR TITLE
SNOW-665361: Bumped up PythonConnector PATCH version from 2.7.12 to 2…

### PR DESCRIPTION
**Pull request by sfc-gh-kdama**
….7.13

@noreview - This is an automated process. No review is required

Please answer these questions before submitting your pull requests. Thanks!

1. What GitHub issue is this PR addressing? Make sure that there is an accompanying issue to your PR.

   Fixes #NNNN

2. Fill out the following pre-review checklist:

   - [ ] I am adding a new automated test(s) to verify correctness of my new code
   - [ ] I am adding new logging messages
   - [ ] I am adding a new telemetry message
   - [ ] I am modifying authorization mechanisms
   - [ ] I am adding new credentials
   - [ ] I am modifying OCSP code
   - [ ] I am adding a new dependency

3. Please describe how your code solves the related issue.

   Please write a short description of how your code change solves the related issue.

### CODE REVIEW

**Comment by sfc-gh-mkeller** [comment]
CC @sfc-gh-sfan I was wrong 😞 

There were other commits in this release, so I had to revert [`c2ea731` (#1255)](https://github.com/snowflakedb/snowflake-connector-python/pull/1255/commits/c2ea731c397efd29d1aa2dec12683cb03eee57e4)

**Review by sfc-gh-sfan — APPROVED** [review]
It's okay. I'll just merge this again once the non-BCR is out.

File: DESCRIPTION.md [hunk 1]
```diff
@@ -8,8 +8,13 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
-- v2.7.13 (Unreleased)
+- v2.7.13(September 22,2022)
```
Comment (sfc-gh-mkeller) [inline DESCRIPTION.md:11]: @sfc-gh-kdama should we update this date?

Comment (sfc-gh-mkeller) [reply DESCRIPTION.md:11]: @sfc-gh-sfan let's minor bump because of BCR?
```suggestion
- v2.8.0(September 22,2022)
```

Comment (sfc-gh-sfan) [reply DESCRIPTION.md:11]: Yeah sounds good.

File: tested_requirements/requirements_310.reqs [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 pycryptodomex==3.15.0
-PyJWT==2.4.0
+PyJWT==2.5.0
 pyOpenSSL==22.0.0
 pytz==2022.2.1
 requests==2.28.1
 typing_extensions==4.3.0
 urllib3==1.26.12
-snowflake-connector-python==2.7.12
+snowflake-connector-python==2.7.13
```
Comment (sfc-gh-mkeller) [inline tested_requirements/requirements_310.reqs:18]:
```suggestion
snowflake-connector-python==2.8.0
```

File: tested_requirements/requirements_37.reqs [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 pycryptodomex==3.15.0
-PyJWT==2.4.0
+PyJWT==2.5.0
 pyOpenSSL==22.0.0
 pytz==2022.2.1
 requests==2.28.1
 typing_extensions==4.3.0
 urllib3==1.26.12
-snowflake-connector-python==2.7.12
+snowflake-connector-python==2.7.13
```
Comment (sfc-gh-mkeller) [inline tested_requirements/requirements_37.reqs:18]:
```suggestion
snowflake-connector-python==2.8.0
```

File: tested_requirements/requirements_38.reqs [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
 pycryptodomex==3.15.0
-PyJWT==2.4.0
+PyJWT==2.5.0
 pyOpenSSL==22.0.0
 pytz==2022.2.1
 requests==2.28.1
 typing_extensions==4.3.0
 urllib3==1.26.12
-snowflake-connector-python==2.7.12
+snowflake-connector-python==2.7.13
```
Comment (sfc-gh-mkeller) [inline tested_requirements/requirements_38.reqs:18]:
```suggestion
snowflake-connector-python==2.8.0
```

File: tested_requirements/requirements_39.reqs [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
 pycryptodomex==3.15.0
-PyJWT==2.4.0
+PyJWT==2.5.0
 pyOpenSSL==22.0.0
 pytz==2022.2.1
 requests==2.28.1
 typing_extensions==4.3.0
 urllib3==1.26.12
-snowflake-connector-python==2.7.12
+snowflake-connector-python==2.7.13
```
Comment (sfc-gh-mkeller) [inline tested_requirements/requirements_39.reqs:18]:
```suggestion
snowflake-connector-python==2.8.0
```

**Review by sfc-gh-mkeller — APPROVED** [review]
:ship: :ship:

**Pull request merged**